diff --git a/index.php b/index.php
index d7dc018..0a23184 100644
--- a/index.php
+++ b/index.php
@@ -5,7 +5,10 @@
    in.  Mostly what it does is check the user permissions
    to see what they have access to.
 */
+
+include_once "../../mainfile.php";
 define('IN_FS', true);
+include XOOPS_ROOT_PATH . '/header.php';
 
 require_once(dirname(__FILE__).'/header.php');
 
@@ -165,5 +168,6 @@ if(isset($_SESSION)) {
     }
     unset($_SESSION['ERROR'], $_SESSION['SUCCESS']);
 }
+        include XOOPS_ROOT_PATH . "/footer.php";
 
-?>
\ No newline at end of file
+?>
