--- distfiles/includes/constants.inc.php	2009-04-23 22:56:40.000000000 +0800
+++ ./includes/constants.inc.php	2010-06-16 19:44:04.560058858 +0800
@@ -10,6 +10,13 @@
 
 // Change this line if you move flyspray.conf.php elsewhere
 $conf = @parse_ini_file(Flyspray::get_config_path(), true);
+$conf['database']['dbtype'] = XOOPS_DB_TYPE;
+$conf['database']['dbhost'] = XOOPS_DB_HOST;
+$conf['database']['dbname'] = XOOPS_DB_NAME;
+$conf['database']['dbuser'] = XOOPS_DB_USER;
+$conf['database']['dbpass'] = XOOPS_DB_PASS;
+$dbprefix = $conf['database']['dbprefix'];
+$conf['database']['dbprefix'] =isset($dbprefix) ? XOOPS_DB_PREFIX ."_". $dbprefix : XOOPS_DB_PREFIX;
 
 // $baseurl
 // htmlspecialchars because PHP_SELF is user submitted data, and can be used as an XSS vector.
@@ -84,4 +91,4 @@
 //define('JABBER_DEBUG', true);
 //define('JABBER_DEBUG_FILE''/path/to/my/debug/file');
 //define('FS_MAIL_LOGFILE', BASEDIR . '/logs/maillog.txt');
-?>
\ 文件尾没有 newline 字符
+?>
