diff --git a/templates/loginbox.tpl b/templates/loginbox.tpl
index 7bef017..4711a02 100644
--- a/templates/loginbox.tpl
+++ b/templates/loginbox.tpl
@@ -1,13 +1,13 @@
 
 <form id="login" action="{$baseurl}index.php?do=authenticate" method="post">
 <div>
-  <label for="lbl_user_name">{L('username')}</label>
+  <label class="inline" for="lbl_user_name">{L('username')}</label>
   <input class="text" type="text" id="lbl_user_name" name="user_name" size="17" maxlength="30" />
 
-  <label for="lbl_password">{L('password')}</label>
+  <label class="inline" for="lbl_password">{L('password')}</label>
   <input class="password" type="password" id="lbl_password" name="password" size="17" maxlength="30" />
 
-  <label for="lbl_remember">{L('rememberme')}</label>
+  <label class="inline" for="lbl_remember">{L('rememberme')}</label>
   <input type="checkbox" id="lbl_remember" name="remember_login" />
   
   <input type="hidden" name="return_to" value="{$_SERVER['REQUEST_URI']}" />
